TodoApp/TodoApp.Api/Models/Item.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Web;
+
+namespace TodoApp.Api.Models
+{
+    public class Item
+    {
+        [Key]
+        public string Id { get; set; }
+
+        [MinLength(1, ErrorMessage = "Text length must be greater than 1.")]
```

Comment:
This is the validation of the text so it should be tested. After discussion with Petr, we would prefer removing of these notations and writing your own validator later (together with business logic).